packages/flutter/lib/src/animation/scroll_behavior.dart

```diff
@@ -78,6 +77,17 @@ class BoundedBehavior extends ExtentScrollBehavior {
   }
 }
 
+Simulation _createFlingScrollSimulation(double position, double velocity, double minScrollOffset, double maxScrollOffset) {
```

Comment:
There's a typo in this file (in code you didn't touch) -- "obatin" instead of "obtain". If you could fix it while you're here that'd be great. (Also all those /// comments should end with a period.)